specs/cm/change-mgt-shapes.ttl

```diff
@@ -20,6 +20,19 @@
 @prefix dcterms: <http://purl.org/dc/terms/> .
 @prefix xsd:     <http://www.w3.org/2001/XMLSchema#> .
 @prefix oslc_rm: <http://open-services.net/ns/rm#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+<http://open-services.net/ns/cm/shapes/3.0/>
+  a oslc:ResourceShapeConstraints ;
+  dcterms:title "The OSLC Change Management(CM) Constraints" ;
+  rdfs:label "Change Management(CM)" ;
+  dcterms:description "Constraints on vocabulary terms defined in the OSLC Change Management (CM) namespace."^^rdf:XMLLiteral ;
+  dcterms:publisher <https://open-services.net/about/> ;
+  dcterms:issued "2020-08-06"^^<http://www.w3.org/2001/XMLSchema#date> ;
+  dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
+  dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/cm/change-mgt-shapes.ttl> ;
+  dcterms:isPartOf <https://oslc-op.github.io/oslc-specs/specs/cm/change-mgt-spec.html> ;
+  dcterms:hasVersion "PS01" .
 
 
 <#ChangeRequestShape> a oslc:ResourceShape ;
```

Comment:
All the resources should have the same base here.

I suggest `@prefix : <http://open-services.net/ns/cm/shapes/3.0/#> . ` and then `: a oslc:ResourceShapeConstraints` and `:ChangeRequestShape a oslc:ResourceShape`.

Reply:
The oslc-core spec does not have the # and /# seems strange. Maybe we should have said .../3.0#, not .../3.0/ for proper fragments. But I'm not sure fragments matter in a .ttl file.

Reply:
ShapeChecker didn't like not having the fragment. So I took off the /, as in .../3.0#.